src/capability/msix.rs

```diff
@@ -39,6 +39,28 @@ impl MsixCapability {
         }
     }
 
+    pub fn get_enabled(&self, access: impl ConfigRegionAccess) -> bool {
```

Comment:
Could we go with the standard Rust getter convention here and just call this `enabled`?

Reply:
Done